onnxruntime/contrib_ops/webgpu/quantization/dp4a_matmul.wgsl.template

```diff
@@ -4,6 +4,7 @@
 #param block_size
 #param n_bits
 #param has_zero_points
+#param is_qualcomm
```

Comment:
signed off but wondering, can this be made as generic sg_size == 64 support, without is_qualcomm.

the 

else
        {
            for (var i = 0u; i < 16u; i++)
            {
                lane_outputs[i] += SDP8AI(own_a0, tile_B[0][base_B + i], own_a1, tile_B[1][base_B + i],  own_scale_a * scale_B[base_B + i], zeroes[base_B + i]);
            }
        }
        
blocks can be eliminated that way.